lib/super_digit.rb

```diff
@@ -1,15 +1,30 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - best case scenario, where n has only 1 digit, is O(1) - constant; otherwise is O(n), where n = number of digits of the integer
+# Space Complexity - O(n), where n = number of digits of the integer
 def super_digit(n)
```

Comment:
The summing is O(n) where n is the number of digits.  Since you're doing recursion n times, this is O(n^2)